tests/admin/class-paper-presenter-test.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace Yoast\WP\Free\Tests\Admin;
+
+use Yoast\WP\Free\Tests\TestCase;
+use \WPSEO_Paper_Presenter;
+use Brain\Monkey;
+
+/**
+ * Class WPSEO_Paper_Presenter_Test
+ *
+ * @package Yoast\WP\Free\Tests\Admin
+ */
+class WPSEO_Paper_Presenter_Test extends TestCase {
+
+	/**
+	 * Tests whether the \WPSEO_Paper_Presenter can be used with a content option instead of a view file.
+	 *
+	 * @covers WPSEO_Paper_Presenter::__construct
```

Comment:
I think the docblock variants should always be FQCNs. And should not be the reason to `use` them on top.

```suggestion
	 * @covers \WPSEO_Paper_Presenter::__construct
```

Also the case for the other `@covers` statements.